engine/service/http.go

```diff
@@ -75,7 +75,7 @@ func WriteJSON(w http.ResponseWriter, data interface{}, status int) error {
 	if e != nil {
 		return sdk.WrapError(e, "WriteJSON> unable to marshal : %v", e)
```

Comment:
well the message if not very useful here

Reply:
Yes and the err should not be printed inside, I will look a this.